flink-examples/src/main/java/io/pravega/examples/flink/alert/AccessLog.java

```diff
@@ -83,15 +94,19 @@ public int hashCode() {
         final int prime = 31;
         int result = 1;
         result = prime * result + ((Status == null) ? 0 : Status.hashCode());
-        result = prime * result + (int) (Timestamp ^ (Timestamp >>> 32));
         result = prime * result + ((ClientIP == null) ? 0 : ClientIP.hashCode());
+        result = prime * result + ((TimestampStr == null) ? 0 : TimestampStr.hashCode());
         result = prime * result + ((Verb == null) ? 0 : Verb.hashCode());
         return result;
     }
 
     @Override
     public String toString() {
-        Gson gson = new Gson();
-        return gson.toJson(this);
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
I think we could either create a static mapper instance once and make use of it or just derive the result string by concatenating the field values.

Reply:
Done. Used a static mapper instance.